include/JournallingObject.h

```diff
@@ -87,6 +87,8 @@ class LMMS_EXPORT JournallingObject : public SerializingObject
 		return m_journallingStateStack.isEmpty();
 	}
 
+	constexpr static auto JO_NODE_NAME = "journallingObject";
```

Comment:
```suggestion
	constexpr static auto NodeName = "journallingObject";
```
Following the code conventions for public static constants. Removed the "JO" since it's redundant